mlflow/langchain/__init__.py

```diff
@@ -56,6 +56,9 @@
 logger = logging.getLogger(mlflow.__name__)
 
 FLAVOR_NAME = "langchain"
+LOADER_FN_KEY = "_mlflow_loader_fn"
```

Comment:
Is this like a reference to the loader function?

Reply:
I see, we store a function in this key.

Reply:
How is this functio serialized/deserialized?

Reply:
This is wrapped in a mlflow.pyfunc.PythonModel load_context() function (L106 below).
https://github.com/mlflow/mlflow/blob/2167a88a1c2aaec9d62bcb1d96c10c80e9966323/mlflow/langchain/__init__.py#L102-L112

Reply:
I changed to the new implementation to serialize the function using cloudpickle.